scripts/clean_results_csv.py

```diff
@@ -20,6 +27,10 @@ def get_args():
     parser.add_argument(
         "-o", "--output_dir", required=True, help="Path to output directory"
     )
+    parser.add_argument("-d", "--dimer", action="store_true", default=False)
+    parser.add_argument(
+        "-s", "--keep_structure_source", action="store_true", default=False
+    )
```

Comment:
I think it would be helpful to have a `help` field for both of these, just to make it a bit more obvious what they do